README.md

```diff
@@ -252,6 +252,7 @@ base_resource_update {
 | gatk_command_mem_diff | yes | nextflow.util.MemoryUnit | How much to subtract from the task's allocated memory where the remainder is the Java heap max. (should not be changed unless task fails for memory related reasons) |
 | scatter_count | yes | int | Number of intervals to split the desired interval into. Mutect2 will call each interval seperately. |
 | germline_resource_gnomad_vcf | no | path | A stripped down version of the [gnomAD VCF](https://gnomad.broadinstitute.org/) stripped of all unneeded INFO fields, keeping only AF, currently available for GRCh38:`/hot/ref/tool-specific-input/GATK/GRCh38/af-only-gnomad.hg38.vcf.gz` and GRCh37: `/hot/ref/tool-specific-input/GATK/GRCh37/af-only-gnomad.raw.sites.vcf`. |
+| panel_of_normals_vcf | no | path | VCF file of sites observed in normal. Currently available for GRCh38: `/hot/ref/tool-specific-input/GATK/GRCh38/1000g_pon.hg38.vcf.gz`. This could be useful for tumor only mode. |
```

Comment:
Just a thought but we might want to consider creating REAMDE for internal use as this repo is public and external users won't have access to the path. I'm not suggesting a change in this PR.

Reply:
I see your point, but having a second readme can be complicated. How about we change it to this:

```suggestion
| panel_of_normals_vcf | no | path | VCF file of sites observed in normal. This could be useful for tumor only mode. For UCLA-CDS users, currently available for GRCh38: `/hot/ref/tool-specific-input/GATK/GRCh38/1000g_pon.hg38.vcf.gz`. |
```

Or we can pull all those file paths to a separate section of "**UCLA-CDS Reference Files**" in the README

Reply:
I didn't mean to duplicate the README. 

> or we can pull all those file paths to a separate section of "UCLA-CDS Reference Files" in the README

Yeah, I think this makes more sense to me. Perhaps, we want to look at other public pipeline repos and standardize the format. This isn't urgent or anything. I just wanted to raise awareness.

Reply:
This makes sense to use, with some sort of central resource for internal reference files and such

Reply:
I created an issue for this. There are quite some reference files so probably better to be handled together in another PR.